scripts/test.sh

```diff
@@ -1,34 +1,20 @@
 #!/usr/bin/env sh
-
 set -e
 
-`dirname "${0}"`/build.sh
-
-
-cd zk.js
-yarn test
-sleep 1
-cd ..
-
-cd system-programs
-yarn test
-cd ..
-
-
-cd cli
-yarn test
-cd ..
-
-cd relayer
-yarn test
-cd ..
+#npx nx test @lightprotocol/prover.js
```

Comment:
If the last command works, then maybe let's remove the commented ones :)

Reply:
@sergeytimoshin was the intention of the comments a guide how to use it?
If so we could move these to the readme.

Reply:
@ananas-block do these commands really deserve to be in the readme? I tend to just delete them